src/api/flash/FocusCommands.js

```diff
@@ -30,6 +30,14 @@ export function FocusCommands(options) {
     let port = focus._port;
```

Comment:
Not for this PR, but we access focus._port all over the place, even though it's nominally "private" - In most code, I bet we should have an accessor in ActiveDevice